cmd/node/main.go

```diff
@@ -1068,11 +1097,14 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	}
 
 	log.Trace("creating nodes coordinator")
-	if ctx.IsSet(keepOldEpochsData.Name) {
-		generalConfig.StoragePruning.CleanOldEpochsData = !ctx.GlobalBool(keepOldEpochsData.Name)
-	}
-	if ctx.IsSet(numEpochsToSave.Name) {
-		generalConfig.StoragePruning.NumEpochsToKeep = ctx.GlobalUint64(numEpochsToSave.Name)
+
+	if !generalConfig.StoragePruning.FullArchive {
```

Comment:
I think this code can be merged with the new code from L657-L675

Reply:
Done.